lib/realtime/api/tenant.ex

```diff
@@ -20,7 +20,7 @@ defmodule Realtime.Api.Tenant do
     field(:max_concurrent_users, :integer, default: 200)
     field(:max_events_per_second, :integer, default: 100)
     field(:max_bytes_per_second, :integer, default: 100_000)
-    field(:max_channels_per_client, :integer, default: 100)
+    field(:max_channels_per_client, :integer, default: Application.get_env(:realtime, :tenant_max_channels_per_client, 100))
```

Comment:
```suggestion
    field(:max_channels_per_client, :integer, default: Application.get_env(:realtime, :tenant_max_channels_per_client))
```

sorry for the nitpick... 😓

Reply:
no worries! Just pulled.